[2024-10-16T18:06:44.910+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T18:06:44.935+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2024-10-16T18:06:44.951+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T18:06:44.958+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2024-10-16T18:06:44.960+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]> from DB
[2024-10-16T18:06:44.980+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-10-16T18:06:44.985+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T18:06:44.985+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-10-16T18:06:44.996+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T18:06:44.997+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]>
[2024-10-16T18:06:44.997+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-10-16T18:06:44.999+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T18:06:45.004+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T18:06:45.009+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-10-16T18:06:45.009+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [queued]>
[2024-10-16T18:06:45.010+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-10-16T18:06:45.021+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): test.dbt_btc_transform.stg_kaggle__btcusdt> on 2024-10-16 18:06:30.304200+00:00
[2024-10-16T18:06:45.024+0000] {standard_task_runner.py:57} INFO - Started process 1022 to run task
[2024-10-16T18:06:45.028+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'run_dbt_models', 'test.dbt_btc_transform.stg_kaggle__btcusdt', 'manual__2024-10-16T18:06:30.304200+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt_models.py', '--cfg-path', '/tmp/tmphxmi6_ud']
[2024-10-16T18:06:45.028+0000] {standard_task_runner.py:85} INFO - Job 69: Subtask test.dbt_btc_transform.stg_kaggle__btcusdt
[2024-10-16T18:06:45.029+0000] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0x7f7ca9505550>]
[2024-10-16T18:06:45.084+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T18:06:45.092+0000] {task_command.py:410} INFO - Running <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [running]> on host 888d2c584607
[2024-10-16T18:06:45.092+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 1022)
[2024-10-16T18:06:45.093+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 1022)
[2024-10-16T18:06:45.093+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2024-10-16T18:06:45.095+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [running]> from DB
[2024-10-16T18:06:45.139+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T18:06:45.148+0000] {taskinstance.py:868} DEBUG - Clearing XCom data
[2024-10-16T18:06:45.180+0000] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-10-16T18:06:45.196+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_models' AIRFLOW_CTX_TASK_ID='test.dbt_btc_transform.stg_kaggle__btcusdt' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T18:06:30.304200+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-16T18:06:30.304200+00:00'
[2024-10-16T18:06:45.197+0000] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2024-10-16T18:06:45.198+0000] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2024-10-16T18:06:45.198+0000] {bash.py:186} DEBUG - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_models' AIRFLOW_CTX_TASK_ID='test.dbt_btc_transform.stg_kaggle__btcusdt' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T18:06:30.304200+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-16T18:06:30.304200+00:00'
[2024-10-16T18:06:45.199+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-10-16T18:06:45.199+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            export DBT_SCHEMA=analytics &&\n            cd /opt/airflow/dbt &&\n            /home/airflow/.local/bin/dbt --no-write-json test --target dev --models stg_kaggle__btcusdt\n            ']
[2024-10-16T18:06:45.208+0000] {subprocess.py:86} INFO - Output:
[2024-10-16T18:06:47.556+0000] {subprocess.py:93} INFO - [0m18:06:47  Running with dbt=1.8.7
[2024-10-16T18:06:47.912+0000] {subprocess.py:93} INFO - [0m18:06:47  Registered adapter: ***=1.8.2
[2024-10-16T18:06:48.480+0000] {subprocess.py:93} INFO - [0m18:06:48  Found 2 models, 18 data tests, 1 source, 540 macros
[2024-10-16T18:06:48.486+0000] {subprocess.py:93} INFO - [0m18:06:48
[2024-10-16T18:06:48.747+0000] {subprocess.py:93} INFO - [0m18:06:48  Concurrency: 4 threads (target='dev')
[2024-10-16T18:06:48.750+0000] {subprocess.py:93} INFO - [0m18:06:48
[2024-10-16T18:06:48.778+0000] {subprocess.py:93} INFO - [0m18:06:48  1 of 18 START test accepted_values_stg_kaggle__btcusdt_hour_end__True__False ... [RUN]
[2024-10-16T18:06:48.780+0000] {subprocess.py:93} INFO - [0m18:06:48  2 of 18 START test accepted_values_stg_kaggle__btcusdt_hour_start__True__False . [RUN]
[2024-10-16T18:06:48.782+0000] {subprocess.py:93} INFO - [0m18:06:48  3 of 18 START test not_null_stg_kaggle__btcusdt_close .......................... [RUN]
[2024-10-16T18:06:48.784+0000] {subprocess.py:93} INFO - [0m18:06:48  4 of 18 START test not_null_stg_kaggle__btcusdt_close_time ..................... [RUN]
[2024-10-16T18:06:49.069+0000] {subprocess.py:93} INFO - [0m18:06:49  3 of 18 PASS not_null_stg_kaggle__btcusdt_close ................................ [[32mPASS[0m in 0.28s]
[2024-10-16T18:06:49.074+0000] {subprocess.py:93} INFO - [0m18:06:49  4 of 18 PASS not_null_stg_kaggle__btcusdt_close_time ........................... [[32mPASS[0m in 0.28s]
[2024-10-16T18:06:49.081+0000] {subprocess.py:93} INFO - [0m18:06:49  1 of 18 PASS accepted_values_stg_kaggle__btcusdt_hour_end__True__False ......... [[32mPASS[0m in 0.29s]
[2024-10-16T18:06:49.083+0000] {subprocess.py:93} INFO - [0m18:06:49  5 of 18 START test not_null_stg_kaggle__btcusdt_high ........................... [RUN]
[2024-10-16T18:06:49.087+0000] {subprocess.py:93} INFO - [0m18:06:49  6 of 18 START test not_null_stg_kaggle__btcusdt_hour_end ....................... [RUN]
[2024-10-16T18:06:49.094+0000] {subprocess.py:93} INFO - [0m18:06:49  7 of 18 START test not_null_stg_kaggle__btcusdt_hour_start ..................... [RUN]
[2024-10-16T18:06:49.117+0000] {subprocess.py:93} INFO - [0m18:06:49  2 of 18 PASS accepted_values_stg_kaggle__btcusdt_hour_start__True__False ....... [[32mPASS[0m in 0.33s]
[2024-10-16T18:06:49.143+0000] {subprocess.py:93} INFO - [0m18:06:49  8 of 18 START test not_null_stg_kaggle__btcusdt_low ............................ [RUN]
[2024-10-16T18:06:49.266+0000] {subprocess.py:93} INFO - [0m18:06:49  5 of 18 PASS not_null_stg_kaggle__btcusdt_high ................................. [[32mPASS[0m in 0.18s]
[2024-10-16T18:06:49.272+0000] {subprocess.py:93} INFO - [0m18:06:49  9 of 18 START test not_null_stg_kaggle__btcusdt_number_of_trades ............... [RUN]
[2024-10-16T18:06:49.313+0000] {subprocess.py:93} INFO - [0m18:06:49  7 of 18 PASS not_null_stg_kaggle__btcusdt_hour_start ........................... [[32mPASS[0m in 0.21s]
[2024-10-16T18:06:49.315+0000] {subprocess.py:93} INFO - [0m18:06:49  6 of 18 PASS not_null_stg_kaggle__btcusdt_hour_end ............................. [[32mPASS[0m in 0.22s]
[2024-10-16T18:06:49.324+0000] {subprocess.py:93} INFO - [0m18:06:49  8 of 18 PASS not_null_stg_kaggle__btcusdt_low .................................. [[32mPASS[0m in 0.17s]
[2024-10-16T18:06:49.332+0000] {subprocess.py:93} INFO - [0m18:06:49  10 of 18 START test not_null_stg_kaggle__btcusdt_open .......................... [RUN]
[2024-10-16T18:06:49.334+0000] {subprocess.py:93} INFO - [0m18:06:49  11 of 18 START test not_null_stg_kaggle__btcusdt_open_date ..................... [RUN]
[2024-10-16T18:06:49.340+0000] {subprocess.py:93} INFO - [0m18:06:49  12 of 18 START test not_null_stg_kaggle__btcusdt_open_time ..................... [RUN]
[2024-10-16T18:06:49.445+0000] {subprocess.py:93} INFO - [0m18:06:49  9 of 18 PASS not_null_stg_kaggle__btcusdt_number_of_trades ..................... [[32mPASS[0m in 0.17s]
[2024-10-16T18:06:49.453+0000] {subprocess.py:93} INFO - [0m18:06:49  13 of 18 START test not_null_stg_kaggle__btcusdt_quote_asset_volume ............ [RUN]
[2024-10-16T18:06:49.524+0000] {subprocess.py:93} INFO - [0m18:06:49  10 of 18 PASS not_null_stg_kaggle__btcusdt_open ................................ [[32mPASS[0m in 0.19s]
[2024-10-16T18:06:49.533+0000] {subprocess.py:93} INFO - [0m18:06:49  12 of 18 PASS not_null_stg_kaggle__btcusdt_open_time ........................... [[32mPASS[0m in 0.19s]
[2024-10-16T18:06:49.540+0000] {subprocess.py:93} INFO - [0m18:06:49  14 of 18 START test not_null_stg_kaggle__btcusdt_taker_buy_base_asset_volume ... [RUN]
[2024-10-16T18:06:49.542+0000] {subprocess.py:93} INFO - [0m18:06:49  11 of 18 PASS not_null_stg_kaggle__btcusdt_open_date ........................... [[32mPASS[0m in 0.20s]
[2024-10-16T18:06:49.548+0000] {subprocess.py:93} INFO - [0m18:06:49  15 of 18 START test not_null_stg_kaggle__btcusdt_taker_buy_quote_asset_volume .. [RUN]
[2024-10-16T18:06:49.562+0000] {subprocess.py:93} INFO - [0m18:06:49  16 of 18 START test not_null_stg_kaggle__btcusdt_volume ........................ [RUN]
[2024-10-16T18:06:49.609+0000] {subprocess.py:93} INFO - [0m18:06:49  13 of 18 PASS not_null_stg_kaggle__btcusdt_quote_asset_volume .................. [[32mPASS[0m in 0.15s]
[2024-10-16T18:06:49.626+0000] {subprocess.py:93} INFO - [0m18:06:49  17 of 18 START test unique_stg_kaggle__btcusdt_close_time ...................... [RUN]
[2024-10-16T18:06:49.737+0000] {subprocess.py:93} INFO - [0m18:06:49  14 of 18 PASS not_null_stg_kaggle__btcusdt_taker_buy_base_asset_volume ......... [[32mPASS[0m in 0.19s]
[2024-10-16T18:06:49.745+0000] {subprocess.py:93} INFO - [0m18:06:49  18 of 18 START test unique_stg_kaggle__btcusdt_open_time ....................... [RUN]
[2024-10-16T18:06:49.779+0000] {subprocess.py:93} INFO - [0m18:06:49  15 of 18 PASS not_null_stg_kaggle__btcusdt_taker_buy_quote_asset_volume ........ [[32mPASS[0m in 0.23s]
[2024-10-16T18:06:49.791+0000] {subprocess.py:93} INFO - [0m18:06:49  16 of 18 PASS not_null_stg_kaggle__btcusdt_volume .............................. [[32mPASS[0m in 0.22s]
[2024-10-16T18:06:50.068+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [running]> from DB
[2024-10-16T18:06:50.093+0000] {job.py:213} DEBUG - [heartbeat]
[2024-10-16T18:06:50.817+0000] {subprocess.py:93} INFO - [0m18:06:50  18 of 18 FAIL 8 unique_stg_kaggle__btcusdt_open_time ........................... [[31mFAIL 8[0m in 1.07s]
[2024-10-16T18:06:50.970+0000] {subprocess.py:93} INFO - [0m18:06:50  17 of 18 FAIL 8 unique_stg_kaggle__btcusdt_close_time .......................... [[31mFAIL 8[0m in 1.33s]
[2024-10-16T18:06:50.992+0000] {subprocess.py:93} INFO - [0m18:06:50
[2024-10-16T18:06:50.992+0000] {subprocess.py:93} INFO - [0m18:06:50  Finished running 18 data tests in 0 hours 0 minutes and 2.50 seconds (2.50s).
[2024-10-16T18:06:50.998+0000] {subprocess.py:93} INFO - [0m18:06:50
[2024-10-16T18:06:50.999+0000] {subprocess.py:93} INFO - [0m18:06:50  [31mCompleted with 2 errors and 0 warnings:[0m
[2024-10-16T18:06:51.000+0000] {subprocess.py:93} INFO - [0m18:06:51
[2024-10-16T18:06:51.001+0000] {subprocess.py:93} INFO - [0m18:06:51  [31mFailure in test unique_stg_kaggle__btcusdt_open_time (models/staging/kaggle/docs/stg_kaggle__btcusdt.yml)[0m
[2024-10-16T18:06:51.002+0000] {subprocess.py:93} INFO - [0m18:06:51    Got 8 results, configured to fail if != 0
[2024-10-16T18:06:51.003+0000] {subprocess.py:93} INFO - [0m18:06:51
[2024-10-16T18:06:51.004+0000] {subprocess.py:93} INFO - [0m18:06:51    compiled code at target/compiled/dbt_btc_transform/models/staging/kaggle/docs/stg_kaggle__btcusdt.yml/unique_stg_kaggle__btcusdt_open_time.sql
[2024-10-16T18:06:51.005+0000] {subprocess.py:93} INFO - [0m18:06:51
[2024-10-16T18:06:51.006+0000] {subprocess.py:93} INFO - [0m18:06:51  [31mFailure in test unique_stg_kaggle__btcusdt_close_time (models/staging/kaggle/docs/stg_kaggle__btcusdt.yml)[0m
[2024-10-16T18:06:51.007+0000] {subprocess.py:93} INFO - [0m18:06:51    Got 8 results, configured to fail if != 0
[2024-10-16T18:06:51.008+0000] {subprocess.py:93} INFO - [0m18:06:51
[2024-10-16T18:06:51.009+0000] {subprocess.py:93} INFO - [0m18:06:51    compiled code at target/compiled/dbt_btc_transform/models/staging/kaggle/docs/stg_kaggle__btcusdt.yml/unique_stg_kaggle__btcusdt_close_time.sql
[2024-10-16T18:06:51.010+0000] {subprocess.py:93} INFO - [0m18:06:51
[2024-10-16T18:06:51.011+0000] {subprocess.py:93} INFO - [0m18:06:51  Done. PASS=16 WARN=0 ERROR=2 SKIP=0 TOTAL=18
[2024-10-16T18:06:51.705+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-10-16T18:06:51.707+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [running]> from DB
[2024-10-16T18:06:51.726+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-10-16T18:06:51.728+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [running]> from DB
[2024-10-16T18:06:51.731+0000] {taskinstance.py:2305} DEBUG - Task Duration set to 6.733603
[2024-10-16T18:06:51.734+0000] {taskinstance.py:1356} DEBUG - Clearing next_method and next_kwargs.
[2024-10-16T18:06:51.734+0000] {taskinstance.py:1345} INFO - Marking task as UP_FOR_RETRY. dag_id=run_dbt_models, task_id=test.dbt_btc_transform.stg_kaggle__btcusdt, execution_date=20241016T180630, start_date=20241016T180644, end_date=20241016T180651
[2024-10-16T18:06:51.757+0000] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2024-10-16T18:06:51.757+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 69 for task test.dbt_btc_transform.stg_kaggle__btcusdt (Bash command failed. The command returned a non-zero exit code 1.; 1022)
[2024-10-16T18:06:51.792+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2024-10-16T18:06:51.856+0000] {dagrun.py:697} DEBUG - number of tis tasks for <DagRun run_dbt_models @ 2024-10-16 18:06:30.304200+00:00: manual__2024-10-16T18:06:30.304200+00:00, state:running, queued_at: 2024-10-16 18:06:30.664597+00:00. externally triggered: True>: 3 task(s)
[2024-10-16T18:06:51.857+0000] {dagrun.py:718} DEBUG - number of scheduleable tasks for <DagRun run_dbt_models @ 2024-10-16 18:06:30.304200+00:00: manual__2024-10-16T18:06:30.304200+00:00, state:running, queued_at: 2024-10-16 18:06:30.664597+00:00. externally triggered: True>: 2 task(s)
[2024-10-16T18:06:51.859+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T18:06:30.304200+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T18:06:51.859+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T18:06:30.304200+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T18:06:51.860+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T18:06:30.304200+00:00 [None]> dependency 'Trigger Rule' PASSED: False, Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=1, skipped=0, failed=0, upstream_failed=0, removed=0, done=1), upstream_task_ids={'model.dbt_btc_transform.stg_kaggle__btcusdt', 'test.dbt_btc_transform.stg_kaggle__btcusdt'}
[2024-10-16T18:06:51.860+0000] {taskinstance.py:1093} DEBUG - Dependencies not met for <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T18:06:30.304200+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=1, skipped=0, failed=0, upstream_failed=0, removed=0, done=1), upstream_task_ids={'model.dbt_btc_transform.stg_kaggle__btcusdt', 'test.dbt_btc_transform.stg_kaggle__btcusdt'}
[2024-10-16T18:06:51.861+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [up_for_retry]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T18:06:51.862+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [up_for_retry]> dependency 'Not In Retry Period' PASSED: False, Task is not ready for retry yet but will be retried automatically. Current date is 2024-10-16T18:06:51.862257+00:00 and task will be retried at 2024-10-16T18:11:51.731239+00:00.
[2024-10-16T18:06:51.862+0000] {taskinstance.py:1093} DEBUG - Dependencies not met for <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T18:06:30.304200+00:00 [up_for_retry]>, dependency 'Not In Retry Period' FAILED: Task is not ready for retry yet but will be retried automatically. Current date is 2024-10-16T18:06:51.862257+00:00 and task will be retried at 2024-10-16T18:11:51.731239+00:00.
[2024-10-16T18:06:51.868+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
